cpp/include/gpu_operations.h

```diff
@@ -44,9 +44,55 @@ namespace Nice {
 template <typename T>
 class GpuOperations {
  public:
+  static cublasStatus_t do_multiply(cublasHandle_t handle, int n,
```

Comment:
Could you move all these wrapper functions to gpu_util.h and gpu_util.cc? One for declaration and the other for definition. I already put all the wrapper functions I used there. You can take it as a reference.
Besides, the function name you used is also inconsistent with the google code style. I would suggest you change them as well. For example, you can change do_multiply to GpuMatrixScale(...) to make more sense based on the cublas or cusolver api interface called